File: scratch/test_register.ts
const name = 'A'.repeat(300);
const response = await fetch('http://localhost:3000/api/users', {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({
        name,
        email: 'testlongname@example.com',
        password: 'password123'
    })
});

console.log(`Status Code: ${response.status}`);
console.log(`Response: ${await response.text()}`);

File: scratch/test_register.py
import requests

name = 'A' * 300
url = 'http://localhost:3000/api/users'
payload = {
    'name': name,
    'email': 'testlongname@example.com',
    'password': 'password123'
}

try:
    response = requests.post(url, json=payload)
    print(f"Status Code: {response.status_code}")
    print(f"Response: {response.text}")
except Exception as e:
    print(f"Error: {e}")
